### DISCUSSION TITLE
Best type of collection for special issue with DOI for each publication?

### DISCUSSION THREAD

**Discussion by alicetiara** [knowledgefutures/pubpub#2135] [Support]
Hi all, I am the editor of the _Bulletin of Technology & Public Life_ (https://citap.pubpub.org/). 

We are planning on publishing the proceedings of a preconference that some of our affiliates ran in May. This will consist of a short essay by the conference organizers and about 25 short papers.

I am trying to determine what sort of collection to set up for this. We want the papers grouped together into some sort of collection, but each paper/chapter would have its own DOI. Is this possible? It doesn't look like we can do this for conference proceedings, special issues, or books. 

Thanks!
Alice Marwick

**Comment by gabestein** [comment]
Hi Alice! Very cool. I would choose the conference proceedings collection type for this use case. And then you can certainly create a DOI for individual Pubs within the collection by visiting the settings page of those Pubs.
